### ISSUE TITLE
NLog to Cloudwatch from non-AWS app (desktop app)

### ISSUE THREAD

**Issue by lellis1936** [aws/aws-logging-dotnet#130]
## The Question
I am using this library in an attempt to centralize logging from a desktop app.   No logs are written there.   Does logging have to originate from an EC2 or other AWS instance?  Trying to determine why nothing is written on AWS Cloudwatch.   Any suggestions or ideas?

Is there any debugging or troubleshooting method I can use to track this down?

Code is as follows:

	  class Program
		{
			static void Main(string[] args)
			{
				var nlog = new NLogWrapper();
				nlog.Configure();
				nlog.Info("test");
				LogManager.Flush();
			}

			public class NLogWrapper
			{

				private readonly Logger logger;

				public NLogWrapper()
				{
					Configure();
					logger = LogManager.GetCurrentClassLogger();
				}

				public void Configure()
				{
					var config = new LoggingConfiguration();
					string accessKey = "XXXXXXXXXXXXXXXXXXXXXXX";
					string secretKey = "XXXXXXXXXXXXXXXXXXXXXXX";
					var awsTarget = new AWSTarget()
					{
						LogGroup = My Log group",
						Region = "us-east-1",
						Credentials = new Amazon.Runtime.BasicAWSCredentials(accessKey, secretKey)

					};
					config.AddTarget("aws", awsTarget);
					config.LoggingRules.Add(new LoggingRule("*", LogLevel.Debug, awsTarget));
					LogManager.Configuration = config;
				}

				public void Info(object message)
				{
					logger.Info(message);
				}

				public void Error(Exception exception)
				{
					logger.Error(exception);
				}
			}
		}
	}

### Environment

* Build Version: 1.5.2
* OS Info: Windows 10
* Build Environment: VS2019
* Targeted .NET Platform:  4.7.2

---

This is a :question: general question

**Comment by snakefoot** [comment]
Have you tried to enable to enable NLog InternalLogger: https://github.com/NLog/NLog/wiki/Internal-Logging ?

**Comment by lellis1936** [comment]
I did.  I've revised my test program (above) to also write to the console as well as the AWS target.  100 test messages are written, and they appear on the console but nothing on AWS.

The resulting log file tells me nothing, but here it is: https://web-widgets.s3.us-east-2.amazonaws.com/log.txt... just in case someone is kind enough to have a look to see if they can spot something.

I am using root AWS credentials for this test.
